src/radical/pilot/configs/resource_ncsa.json

```diff
@@ -15,7 +15,47 @@
         "agent_launch_method"         : "APRUN",
         "task_launch_method"          : "APRUN",
         "mpi_launch_method"           : "APRUN",
-        "pre_bootstrap_1"             : ["module load bw-python"],
+        "pre_bootstrap_1"             : [
+            "module switch PrgEnv-cray PrgEnv-gnu",
+            "module load bwpy",
+            "module use --append /projects/sciteam/gkd/modules",
+            "module load openmpi/rhc_static"
+        ],
+        "default_remote_workdir"      : "/scratch/sciteam/$USER",
+        "valid_roots"                 : ["/scratch/sciteam"],
+        "rp_version"                  : "local",
+        "virtenv"                     : "%(global_sandbox)s/ve_bw",
+        "virtenv_mode"                : "create",
+        "stage_cacerts"               : "True",
+        "python_dist"                 : "default"
+    },
+
+    "bw_lib": {
+        "description"                 : "The NCSA Blue Waters Cray XE6/XK7 system (https://bluewaters.ncsa.illinois.edu/)",
+        "notes"                       : "Running 'touch .hushlogin' on the login node will reduce the likelihood of prompt detection issues.",
+        "schemas"                     : ["gsissh"],
+        "gsissh"                      : {
+            "job_manager_endpoint"    : "torque+gsissh://bw.ncsa.illinois.edu",
+            "filesystem_endpoint"     : "gsisftp://bw.ncsa.illinois.edu/"
+        },
+        "default_queue"               : "normal",
+        "lrms"                        : "TORQUE",
+        "agent_type"                  : "multicore",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "ORTE",
+        "agent_config"                : "cray",
+        "agent_launch_method"         : "ORTE_LIB",
+        "task_launch_method"          : "ORTE_LIB",
+        "mpi_launch_method"           : "ORTE_LIB",
+        "tunnel_bind_device"          : "ipogif0",
```

Comment:
Same as earlier: shouldn't agent_launch_method be `ORTE`, since the POPEN is hardcoded?